CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Guidelines for contribution of new pipelines
+
+> [!TIP]
+> The process of submitting a pipeline for inclusion is done entirely through the review of
+> a pull request. Most of the pre-review checklist is codified in the "New pipeline" issue
+> template. Submissions of new pipelines that do not use this template will not be merged.
+
+## Pre-review checklist
+
+### General information about the pipeline
+
+- [ ] Descriptive title for the pipeline
+- [ ] Authorship information with appropriate CRediT roles
+- [ ] Abstract 
```

Comment:
Is this the markdown description that goes along iwth the pipelines, and that we display on the web page? example: https://boninabox.geobon.org/indicator?i=RLI

If yes, we would probably need another checklist for what the abstract should contain. I am weary that this .md abstract duplicates a lot of the information from the yml description file, and that it goes out of sync. The ideal would be that this page is generated by the yml file's info.